src/test/resources/liquibase/harness/snapshot/expectedSnapshot/redshift/addColumn.json

```diff
@@ -1,48 +1,34 @@
 {
   "snapshot": {
     "objects": {
-      "liquibase.structure.core.Table": [
+      "liquibase.structure.core.Catalog": [
```

Comment:
we don't check Catalogs and Schemas in all tests, only in snapshotCatalogAndSchema.json. But details that matters, like column details are missing